src/SFA.DAS.EmployerAccounts/Queries/GetSingleCohort/GetSingleCohortResponse.cs

```diff
@@ -5,5 +5,6 @@ namespace SFA.DAS.EmployerAccounts.Queries.GetSingleCohort
     public class GetSingleCohortResponse
     {
         public Cohort Cohort { get; set; }
+        public bool HasFailed { get; set; }
```

Comment:
If this is to be common across all response types then it should be put into a base class.